index.js

```diff
@@ -143,8 +172,16 @@ function sortCarInventory(/* code here */) {
  *     (1) an array which is an inventory of cars like the one inside /data/inventory.js.
  * getModelYears returns an array containing all the 'car_year's in the inventory.
 */
-function getModelYears(/* code here */) {
-  /* code here */
+function getModelYears(array) {
+
+  const getModelYears = [];
```

Comment:
to avoid confusion, i wouldn't name my array the same as my function. Instead you could just name is modelYears